src/api/crypto.js

```diff
@@ -112,20 +102,9 @@ class SecEncKey extends h.NetworkObject {
   * @returns {Promise<Buffer>} plain text
   */
   decrypt(cipher, publicEncKey) {
+    if (!publicEncKey) throw new Error(errConst.MISSING_PUB_ENC_KEY.msg);
     return lib.decrypt(this.app.connection, cipher, publicEncKey.ref, this.ref);
   }
-
-  /**
-  * Encrypt the input (buffer or string) using this secret encryption key
-  * and the recipient's public key
-  *
-  * @param {Buffer} data to be encrypted
-  * @param {PubEncKey} recipientPubKey recipient's public encryption key
-  * @returns {Promise<Buffer>} cipher text
-  */
-  encrypt(data, recipientPubKey) {
```

Comment:
Why are we removing this function?

Reply:
Same here: I was confused about this part of the API. Why do `PubEncKey` and `SecEncKey` each have an `encrypt` and `decrypt` function?

I don't see that reflected in web API:
- https://github.com/joshuef/peruse/blob/master/app/extensions/safe/api/crypto_public_encryption_key.js#L3
- https://github.com/joshuef/peruse/blob/master/app/extensions/safe/api/crypto_secret_encryption_key.js#L3

Reply:
Isn't the `encrypt` method redundant in `SecEncKey`, since I can get a recipients public key from raw bytes and then `encrypt` using the `PubEncKey` method?